cfme/tests/services/test_provision_stack.py

```diff
@@ -152,6 +156,18 @@ def stack_created(appliance, provider, order_stack, stack_data):
     yield stack
 
 
+@pytest.fixture
+def stack_not_created_error(appliance, provider, order_stack, stack_data, request):
+    if "test_error_message_azure" in request.node.name:
```

Comment:
I added this `if` in case someone wants to add some other errors that should be checked